src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/MicrosoftIdentityPlatform/MicrosoftIdentityPlatformApplicationManager.cs

```diff
@@ -155,6 +148,31 @@ public class MicrosoftIdentityPlatformApplicationManager
             }
         }
 
+        /// <summary>
+        /// Attempt to add current user as owner of app registration, exception indicates the owner already exists
+        /// </summary>
+        /// <param name="graphServiceClient"></param>
+        /// <param name="createdApplication"></param>
+        /// <returns></returns>
+        internal static async Task AddCurrentUserAsOwner(GraphServiceClient graphServiceClient, Application? createdApplication)
+        {
+            try
+            {
+                // Add the current user as a owner.
+                User? me = await graphServiceClient.Me.GetAsync();
+                var requestBody = new ReferenceCreate
+                {
+                    OdataId = $"https://graph.microsoft.com/beta/directoryObjects/{me?.Id}",
+                };
+
+                await graphServiceClient.Applications[createdApplication!.Id].Owners.Ref.PostAsync(requestBody);
+            }
+            catch (Graph.Models.ODataErrors.ODataError)
+            {
+                // Owner already exists in app registration
```

Comment:
should we do something in this catch, maybe just console output?

Reply:
In this case, I don't think it's worth logging, as the info doesn't really affect the customer and the scenario where the exception is thrown seems to be limited to only the microsoft tenant at this point